src/pages/signup/Signup.tsx

```
@@ -0,0 +1,41 @@
+import styled from 'styled-components'
+import { AuthLayout } from '../../components/AuthLayout'
+import First from './First'
+import { useState } from 'react'
+import Second from './Second'
+import Third from './Third'
+
+function Signup() {
+  const [step, setStep] = useState<number>(2)
```

ESLint

[error] 'setStep' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)